src/Show.jl

```diff
@@ -24,9 +24,25 @@ export status
 
 import Oscar: pretty, Indent, Dedent
 
+"""
```

Comment:
Hint: if you need to use e.g. `$` in a docstring, you can also do this:
```suggestion
@doc raw"""
```

But even without this it should be fine to use `"`